webapp/apps/test_assets/test_reform.py

```diff
@@ -0,0 +1,111 @@
+reform_text = """// Title: 2016 Trump Campaign Tax Plan
+// Reform_File_Author: Matt Jensen
```

Comment:
@brittainhard, what are the pros/cons of copying the test here rather than pulling it directly from taxcalc?